### PR TITLE
Modified dory installer version as v32 and modified documentation in manual installation document

### PR DESCRIPTION
**Pull request by bhagyashree-sarawate**
Hi William,
This is ansible installer change to install the dory v32 version as per the new requirement

Regards
Bhagyashree Sarawate

### CODE REVIEW

**Comment by sneharai4** [comment]
@bhagyashree-sarawate The changes looks fine. But we should add the dory installer variable in our properties file, so that we need to make changes only in prop file instead of ansible playbook.

File: ansible_3par_docker_plugin/README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ These playbooks perform the following tasks on the Master/Worker nodes as define
       | ```hpe3par_iscsi_ips```  | No  | No default value | Comma separated iscsi IPs. If not provided, all iscsi IPs will be read from the array and populated in hpe.conf |
       | ```vlan_tag```  | No  | False | Populates the iscsi_ips which are vlan tagged, only applicable if ```hpe3par_iscsi_ips``` is not specified |
       | ```replication_device```  | No  | No default value | Replication backend properties |
+      | ```dory_installer_version```  | Yes  | dory_installer_v32 | Required for Openshift/Kubernetes setup.Dory installer version, supported versions are dory_installer_v31,dory_installer_v32 |
```
Comment (sneharai4) [inline ansible_3par_docker_plugin/README.md]: @bhagyashree-sarawate Dory installer is not required when you deploy plugin on a standalone docker env.
Should this be still set to mandatory "Yes"?

Rest all changes looks good to me.

Comment (bhagyashree-sarawate) [reply ansible_3par_docker_plugin/README.md]: Done

**Comment by sneharai4** [comment]
@wdurairaj @bhagyashree-sarawate Do we need to update https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/docker%20EE%202.0%20UCP%203.0.5%20installation.md doc with dory_installer_v32?

**Comment by sneharai4** [comment]
@bhagyashree-sarawate Please add the log file for this change. Other than that the changes looks fine.

**Comment by bhagyashree-sarawate** [comment]
[Installer_log_PR688.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3420634/Installer_log_PR688.txt)

Attached installation logs

File: ansible_3par_docker_plugin/README.md [hunk 2]
```diff
@@ -54,6 +54,7 @@ These playbooks perform the following tasks on the Master/Worker nodes as define
       | ```hpe3par_iscsi_ips```  | No  | No default value | Comma separated iscsi IPs. If not provided, all iscsi IPs will be read from the array and populated in hpe.conf |
       | ```vlan_tag```  | No  | False | Populates the iscsi_ips which are vlan tagged, only applicable if ```hpe3par_iscsi_ips``` is not specified |
       | ```replication_device```  | No  | No default value | Replication backend properties |
+      | ```dory_installer_version```  | No  | dory_installer_v32 | Required for Openshift/Kubernetes setup.Dory installer version, supported versions are dory_installer_v31,dory_installer_v32 |
```
Comment (sneharai4) [inline ansible_3par_docker_plugin/README.md]: @bhagyashree-sarawate Please add a space after full stop and comma. Rest looks good to me.

**Pull request merged**